Capri.Web/src/store.ts

```diff
@@ -0,0 +1,18 @@
+﻿import Vue from 'vue';
+import Vuex from 'vuex';
+
+Vue.use(Vuex);
+
+export default new Vuex.Store({
+    state: {
+
+    },
+
+    mutations: {
+
+    },
+
+    actions: {
+
+    }
```

Comment:
I actually woul separate it to modules. It will be cleaner then:
```typescript
export const store = new Vuex.Store({
    modules: {
        login: LoginModule,
        usersList: UsersListModule,
    }
});
```

Modules should be in separate files imported only to main store file (it would be nice to move `store.ts` to `store` directory under `src/`). and then sample module file should look like this:

```typescript
const loginState: State = {

};

export const LoginModule = {
    state: loginState,
    mutations: {

    },
    actions: {

    },

    getters: {

    }
};

```